site2/tools/publish-website.sh

```diff
@@ -65,7 +65,7 @@ PULSAR_SITE_TMP=/tmp/pulsar-site
       | grep -v $PULSAR_SITE_TMP/content/.htaccess \
       | xargs rm -rf
   fi
-  cp -r $GENERATED_SITE_DIR/content/* $PULSAR_SITE_TMP/content
+  cp -r $GENERATED_SITE_DIR/content/ $PULSAR_SITE_TMP/content
```

Comment:
Perhaps using `-R` to avoid behavior inconsistency. For the implementation on a darwin OS:

```
     Historic versions of the cp utility had a -r option.  This implementation
     supports that option, however, its behavior is different from historical
     FreeBSD behavior.  Use of this option is strongly discouraged as the
     behavior is implementation-dependent.  In FreeBSD, -r is a synonym for -RL
     and works the same unless modified by other flags.  Historical
     implementations of -r differ as they copy special files as normal files
     while recreating a hierarchy.
```

Reply:
fixed with https://github.com/apache/pulsar-site/pull/264